tests/unit/style/properties/serialization.rs

```diff
@@ -536,10 +536,11 @@ mod shorthand_serialization {
     fn transition_should_serialize_all_available_properties() {
         use euclid::point::Point2D;
         use style::properties::animated_properties::TransitionProperty;
+        use style::properties::longhands::transition_delay::SpecifiedValue as DelayContainer;
         use style::properties::longhands::transition_duration::SpecifiedValue as DurationContainer;
         use style::properties::longhands::transition_property::SpecifiedValue as PropertyContainer;
         use style::properties::longhands::transition_timing_function::SpecifiedValue as TimingContainer;
-        use style::properties::longhands::transition_timing_function::TransitionTimingFunction;
+        use style::properties::longhands::transition_timing_function::single_value::SpecifiedValue as TransitionTimingFunction;
```

Comment:
I don't think you can. `use style::properties::longhands::transition_timing_function` and then `transition_timing_function::whatever` in the code might work better (replace all TTF imports if you're doing this)